lib/foodsoft_config.rb

```diff
@@ -75,6 +75,11 @@ def init(filename = APP_CONFIG_FILE)
       self.scope = config[:default_scope] or raise "No default_scope is set"
       # Set defaults for backward-compatibility
       set_missing
+      # When multicoops is enabled, the foodcoop is selected on each request.
+      # When multicoops is disabled, we need to do some initialization.
+      unless config[:multi_coop_install]
```

Comment:
do we really need the `unless`. if not, i'd omit it

Reply:
We don't (I'll test) :+1: